### PR TITLE
adding video_conference_url property

### PR DESCRIPTION
**Pull request by joshreisner**
to address #9 

wdyt @tintinnabulate?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ The JSON file is expected to contain a simple array of meetings. [Here is an exa
 
 `venmo` is an optional string and should be a valid Venmo handle, eg @FFG-Group. This is understood to be the address for 7th Tradition contributions to the group, and not any other entity.
 
+`video_conference_url` is an optional URL to a specific public videoconference meeting. This should be a common videoconferencing service such as Zoom or Google Hangouts. It should launch directly into the meeting, without requiring the user to type a meeting ID, password, or be invited to join beforehand. Online meetings should still have a physical address, and an optional meeting type of `TC` (Temporary Closure). This spec is still for geographic meetings; a true online-only spec is in the planning stages.
```
Comment (tintinnabulate) [inline README.md]: Yes I agree with this comment. LGTM.

Sidenote: I didn't know that true online-only spec is in the planning stages - can I subscirbe to get updates on this? How?

Comment (joshreisner) [reply README.md]: Right now it exists in the form of a [Google Sheet here](https://docs.google.com/spreadsheets/d/e/2PACX-1vTTo77YZG5U6uTMSsJQiL9TMlm4w1Assojsswb5ElBv_7ckhw-Z5pbQVRZhwSesvQlodxjQkWbp1bSc/pubhtml). It's the data source for a [new meeting finder for OiAA](https://github.com/code4recovery/oiaa) that I developed a few weeks ago (coincidentally) and they are in the process of implementing. (We're having a meeting tomorrow night at 530 PT if you want to join.)

If this works out for them I was thinking of writing it up as a JSON spec.

Comment (tintinnabulate) [reply README.md]: Yes I would like to join the meeting :)

How do I join?

just to check, is 

> "530 PT"

Is this Tuesday 17:30 PM Pacific Standard Time? Is that the same as Wednesday 12:30 AM UTC

Comment (joshreisner) [reply README.md]: > The next Web Committee Meeting will be Tuesday, March 17, 2020 via Zoom:
>
> EST, 8:30 pm.
> West Coast 5:30 pm.
> Thailand 8:30 am.
> Zoom Meeting ID: 440-227-720

**Comment by FlipperPA** [comment]
Sorry I'm late to the party, @joshreisner and @tintinnabulate, I'm just about to implement some of this in the Wagtail Meeting Guide Django package. The fields I'd like to add are 7th Tradition payment fields for venmo and paypal, video conference URL, and video conference dial-in number. I was thinking:

`video_conference_url`: URLField (validated) https://zoom.com/blahblah/
`video_conference_dial_in`: VarChar (200) (+1-800-123-1234 Meeting ID: 123 456 789)`
`payment_venmo`: @FFG-Group
`payment_paypal`: mypaypalemail@gmail.com

What do you think? I'll be happy to draft a P.R., and sorry again for being late after the merge with these ideas. Life has been a bit chaotic of late.

File: README.md (hunk 1 above)
Comment (tintinnabulate) [reply README.md]: NooOooOoO - I overslept :man_facepalming:

I'm so sorry. The call was at Wednesday 01:30 AM my time and I was fast asleep. On the plus side, I've had the best sleep in days! ... And I'm about to go back to sleep haha.

I'd love to be involved in the discussion of this though. The timezone difference makes this a bit difficult but perhaps I can contribute asynchronously.

**Pull request merged**